nebula-spark-connector/src/main/scala/com/vesoft/nebula/connector/NebulaConfig.scala

```diff
@@ -88,12 +88,17 @@ object NebulaConnectionConfig {
 /**
   * Base config needed when write dataframe into nebula graph
   */
-private[connector] class WriteNebulaConfig(space: String, user: String, passwd: String, batch: Int)
+private[connector] class WriteNebulaConfig(space: String,
+                                           user: String,
+                                           passwd: String,
+                                           batch: Int,
+                                           writeMode: String)
```

Comment:
how about using enum for `writeMode`?

Reply:
Thanks, entry parameter is changed to WriteMode enum.